### ISSUE TITLE
Uploading bug - Interceptor issue

### ISSUE THREAD

**Issue by Ispen** [nestjs/nest#1536]
## I'm submitting a

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Using Interceptors on any endpoint causing a timeout, where request can't reach the body of the function.

## Expected behavior
Request should be handled.

## Minimal reproduction of the problem with instructions

0. You need to have package on version "@nestjs/core": "5.7.0",
With Core 5.6.0 all working great. @nestjs/common can be latest.
1. In your controller add endpoint with
```
@Post('upload')
@UseInterceptors(FilesInterceptor('file'))
async uploadFiles(@UploadedFiles() file) {/*it will never reach here*/}
```
## Environment

<pre><code>
Nest version: 5.7.0

 
For Tooling issues:
- Node version: 8.11.3
- Platform:  Linux

QUICK FIX: use `@nestjs/core: 5.6.0`
</code></pre>

**Comment by uncledlm** [comment]
I had an issue where I updated from 5.6 to the 5.7 and none of the routes returned.  Looks like the issue is with Middlewares. Do you have any Middlewares?

**Comment by Ispen** [comment]
>Do you have any Middlewares?
@uncledlm  Yes, a lot, but everything working perfect, even in that controller. The only exceptions are endpoints with Files Interceptor. Commenting out `@UseInterceptors(FilesInterceptor('file'))` makes this endpoint working.

**Comment by uncledlm** [comment]
Ok, I had a project that had only 1 Middleware (I even replaced the code in it with whats in the docs) and all of my routes stopped working when I upgraded to 5.7.  However, the routes excluded from my Middleware continued to work.  When I downgraded to 5.6 everything worked.

**Comment by kamilmysliwiec** [comment]
Fixed in 5.7.1.

**Comment by Ispen** [comment]
@kamilmysliwiec I confirm, fixed, good job!
